app.yaml.ctmpl

```diff
@@ -0,0 +1,42 @@
+{{with $environment := env "ENVIRONMENT"}}
+{{with $commonSecrets := vault (printf "secret/dsde/firecloud/common/oauth_client_id")}}
+{{with $googleProject := env "GOOGLE_PROJ"}}
+{{with $serviceVersion := env "SERVICE_VERSION"}}
+
+runtime: python27
+threadsafe: true
+api_version: 1
+basic_scaling:
+  max_instances: 2
+
+#[START_EXCLUDE]
+skip_files:
+- ^(.*/)?#.*#$
+- ^(.*/)?.*~$
+- ^(.*/)?.*\.py[co]$
+- ^(.*/)?.*/RCS/.*$
+- ^(.*/)?\..*$
+- ^(.*/)?setuptools/script \(dev\).tmpl$
+#[END_EXCLUDE]
+
+handlers:
+- url: /api/.*
+  script: main.api
+
+libraries:
+- name: pycrypto
+  version: 2.6
+- name: ssl
+  version: 2.7.11
+
+# [START env_vars]
+env_variables:
+  # The following values are to be replaced by information from the output of
+  # 'gcloud endpoints services deploy swagger.json' command.
+  ENDPOINTS_SERVICE_NAME: {{$googleProject}}.appspot.com
+  ENDPOINTS_SERVICE_VERSION: 2018-06-28r6
```

Comment:
this is a temp solution, will parse this out of `gcloud endpoints services deploy` output until google fixes the `describe` command

Reply:
I would templatize this now and feed in the temp value